.gitattributes

```diff
@@ -0,0 +1,30 @@
+# Java sources
```

Comment:
can't we say .bat is CRLF, everything else if LF?

Reply:
Yes, I saw that you can have a default behaviour and use overrides like this : 
* text=auto
*.bat text eol=crlf
I 'll use that